driver/examples/plot_cube.py

```diff
@@ -0,0 +1,22 @@
+import matplotlib.pyplot as plt
```

Comment:
later: If we're going to be plotting cube data a lot, we might consider copy-pasting `pcolormesh_cube` into one of the packages in `pace` so it's easier for users to set up their environment.